xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -1306,31 +1306,31 @@ p.WaitForExit();
-  
+> <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute*> must be `false` if the <xref:System.Diagnostics.ProcessStartInfo.UserName*> property is not `null` or an empty string, or an <xref:System.InvalidOperationException> will be thrown when the <xref:System.Diagnostics.Process.Start%28System.Diagnostics.ProcessStartInfo%29?displayProperty=nameWithType> method is called.
+
+When you use the operating system shell to start processes, you can start any document (which is any registered file type associated with an executable that has a default open action) and perform operations on the file, such as printing, by using the <xref:System.Diagnostics.Process> object. When <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute*> is `false`, you can start only executables by using the <xref:System.Diagnostics.Process> object.
+
 > [!NOTE]
->  <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute%2A> must be `true` if you set the <xref:System.Diagnostics.ProcessStartInfo.ErrorDialog%2A> property to `true`.  
-  
- The <xref:System.Diagnostics.ProcessStartInfo.WorkingDirectory%2A> property behaves differently depending on the value of the <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute%2A> property. When <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute%2A> is `true`, the <xref:System.Diagnostics.ProcessStartInfo.WorkingDirectory%2A> property specifies the location of the executable. If <xref:System.Diagnostics.ProcessStartInfo.WorkingDirectory%2A> is an empty string, it is assumed that the current directory contains the executable.  
-  
- When <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute%2A> is `false`, the <xref:System.Diagnostics.ProcessStartInfo.WorkingDirectory%2A> property is not used to find the executable. Instead, it is used only by the process that is started and has meaning only within the context of the new process. When <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute%2A> is `false`, the <xref:System.Diagnostics.ProcessStartInfo.FileName%2A> property can be either a fully qualified path to the executable, or a simple executable name that the system will attempt to find within folders specified by the PATH environment variable.  
-  
-   
-  
-## Examples  
- [!code-cpp[ProcessOneStream#1](~/samples/snippets/cpp/VS_Snippets_CLR/ProcessOneStream/CPP/stdstr.cpp#1)]
- [!code-csharp[ProcessOneStream#1](~/samples/snippets/csharp/VS_Snippets_CLR/ProcessOneStream/CS/stdstr.cs#1)]
- [!code-vb[ProcessOneStream#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/ProcessOneStream/VB/stdstr.vb#1)]  
-  
- ]]></format>
+> <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute*> must be `true` if you set the <xref:System.Diagnostics.ProcessStartInfo.ErrorDialog*> property to `true`.
+
+If you set the <xref:System.Diagnostics.ProcessWindowStyle> to `Hidden`, <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute*> must be set to `true`.
```

Comment:
I think you mean, "If you set the \<xref:System.Diagnostics.ProcessStartInfo.WindowStyle> property to \<xref:System.Diagnostics.ProcessWindowStyle.Hidden?displayProperty=nameWithType>,"